tests/ValueParsers/DecimalParserTest.php

```diff
@@ -123,4 +123,43 @@ public function testUnlocalization() {
 		$this->assertEquals( '20000000', $value->getValue() );
 	}
 
+	public function provideSplitDecimalExponent() {
```

Comment:
Arrrg on the naming scheme. We agreed on `...Provider`, see the two existing providers in this file. But not a reason to block this patch.